src/providers/oauth-provider.js

```diff
@@ -208,6 +226,20 @@ function OAuthProvider() {
           return response;
         });
       }
+
+      /**
+       * Override default OAuth configuration
+       * @param {object} options - Optional configuration override.
+       * @return {object} A configuration object cleaned.
+       */
+
+      overrideOAuthConfig(options = {}) {
+        const config = Object.keys(defaultConfig).reduce((config, key) => {
```

Comment:
true, I was first trying limit the copy of supported key but it seems to be useless.